### ISSUE TITLE
Compilation fails on Linux for ESP32

### ISSUE THREAD

**Issue by end2endzone** [end2endzone/AnyRtttl#7]
Compilation of example _[Basic](https://github.com/end2endzone/AnyRtttl/tree/0a5bbfb2d35968704c4772a772ab0739481689a8/examples/Basic)_ fails on Linux builds running on Github Actions.

The following message is displayed in the logs:
```
==========================================================================================================
Compiling /home/runner/work/AnyRtttl/AnyRtttl/examples/Basic/Basic.ino (esp8266)
==========================================================================================================
/home/runner/.arduino15/packages/esp8266/hardware/esp8266/3.1.2/tools/elf2bin.py:54: SyntaxWarning: invalid escape sequence '\s'
  words = re.split('\s+', line)
/home/runner/.arduino15/packages/esp8266/hardware/esp8266/3.1.2/tools/elf2bin.py:73: SyntaxWarning: invalid escape sequence '\s'
  words = re.split('\s+', line)
. Variables and constants in RAM (global, static), used 28508 / 80192 bytes (35%)
║   SEGMENT  BYTES    DESCRIPTION
╠══ DATA     1516     initialized variables
╠══ RODATA   1224     constants       
╚══ BSS      25768    zeroed variables
. Instruction RAM (IRAM_ATTR, ICACHE_RAM_ATTR), used 61183 / 65536 bytes (93%)
║   SEGMENT  BYTES    DESCRIPTION
╠══ ICACHE   32768    reserved space for flash instruction cache
╚══ IRAM     28415    code in IRAM    
. Code in flash (default, ICACHE_FLASH_ATTR), used 2[37](https://github.com/end2endzone/AnyRtttl/actions/runs/10256634935/job/28376133449#step:18:38)412 / 1048576 bytes (22%)
║   SEGMENT  BYTES    DESCRIPTION
╚══ IROM     237412   code in flash   

Used library Version Path
AnyRtttl     2.2.1   /home/runner/work/AnyRtttl/AnyRtttl

Used platform   Version Path
esp8266:esp8266 3.1.2   /home/runner/.arduino15/packages/esp8266/hardware/esp8266/3.1.2


==========================================================================================================
Compiling /home/runner/work/AnyRtttl/AnyRtttl/examples/Basic/Basic.ino (esp32)
==========================================================================================================
Traceback (most recent call last):
  File "/home/runner/.arduino15/packages/esp32/tools/esptool_py/4.6/esptool.py", line 34, in <module>
    import esptool
  File "/home/runner/.arduino15/packages/esp32/tools/esptool_py/4.6/esptool/__init__.py", line [41](https://github.com/end2endzone/AnyRtttl/actions/runs/10256634935/job/28376133449#step:18:42), in <module>
    from esptool.cmds import (
  File "/home/runner/.arduino15/packages/esp32/tools/esptool_py/4.6/esptool/cmds.py", line 14, in <module>
    from .bin_image import ELFFile, ImageSegment, LoadFirmwareImage
  File "/home/runner/.arduino15/packages/esp32/tools/esptool_py/4.6/esptool/bin_image.py", line 14, in <module>
    from .loader import ESPLoader
  File "/home/runner/.arduino15/packages/esp32/tools/esptool_py/4.6/esptool/loader.py", line 30, in <module>
    import serial
ModuleNotFoundError: No module named 'serial'
Error during build: exit status 1

Used platform Version Path
esp32:esp32   3.0.3   /home/runner/.arduino15/packages/esp32/hardware/esp32/3.0.3
Error: Process completed with exit code 1.
```

Note that compilation for esp8266 is working fine.

**Comment by end2endzone** [comment]
Other links referencing the same problem:
* https://forum.arduino.cc/t/modulenotfounderror-no-module-named-serial/847838/6
* https://arduino.stackexchange.com/questions/86710/arduino-esp32-import-error-no-module-named-serial
* https://forum.arduino.cc/t/compile-error-modulenotfounderror-no-module-named-serial-solved/1030871
* https://github.com/espressif/arduino-esp32/issues/13


Possible solutions:
* `pip install pyserial`
* `apt install python3-serial`

**Comment by end2endzone** [comment]
Fixed. See https://github.com/end2endzone/AnyRtttl/actions/runs/10258421907
